src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -486,6 +487,29 @@ internal Uri CreateIssuePreview(string projectName, string teamName, IReadOnlyDi
             Uri.TryCreate(finalUrl, UriKind.Absolute, out Uri result);
             return result;
         }
+
+        /// <summary>
+        /// Retains characters within ascii range 1-127 and returns
+        /// an escaped representation of the string
+        /// </summary>
+        /// <param name="str"></param>
+        /// <returns></returns>
+        internal static string EscapeForUrl(string str)
+        {
+            var sb = new StringBuilder();
+            foreach (var c in str)
+            {
+                // characters such as the middle dot (ascii 183) can 
+                // cause issues during navigation, so we only include
+                // a basic set of characters when building the URL
+                if (c >= 1 && c <= 127)
+                {
+                    sb.Append(c);
+                }
```

Comment:
>} [](start = 16, length = 1)

Not sure how I feel about completely removing characters. Would it make sense to have a default substitution character to represent a character that was removed? Since most cases here are likely to be errors in our code or data files, having some sort of way to detect this case would also be interesting.